engine/crates/runtime/src/udf.rs

```diff
@@ -4,8 +4,10 @@ use common_types::UdfKind;
 use serde::Serialize;
 
 #[derive(Debug, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct UdfRequestContextRequest {
     pub headers: serde_json::Value,
+    pub jwt_claims: BTreeMap<String, serde_json::Value>,
```

Comment:
Should we make the field `null` when the request isn't authenticated with a JWT? On the one hand, more context for the custom resolver. On the other hand, it's more work to handle.

Reply:
As opposed to an empty object?

Reply:
yes

Reply:
IMHO it's better, you can't rely on specific claims to be there even if authenticated. if the user needs to know whether he was authenticated through some method or not, we should just add a `is_authenticated` boolean or something like that imho.